reference.md

```diff
@@ -664,33 +662,49 @@ Field Name | Presence | Type | Description
 
 ### wait_time
 
-This dynamic query provides wait time for specific location. To provide wait times to consumers, either this method or `wait_times.json` method can be used. `wait_time` allows more precise queries but requires a call on every interaction by users. 
+This dynamic query provides real time wait times for a specific location. To provide wait times to consumers, either this method or `wait_times.json` method can be used. `wait_time` allows more precise queries but requires a call on every interaction by users.
 
-The request must have the following query parameters. 
+The request must be a POST request (for security reasons) and the body use a JSON format. Here are the following attributes.
```

Comment:
Not opposed to this, myself.

**Pros:**
- reduces the chance of PII being logged on anybody's systems

**Cons:**
- breaking change

```suggestion
The request must be sent as a POST request and the body must use a JSON format with the following attributes.
```

Reply:
Are we the first client to use wait_time?
If so, breaking change is not an issue. Otherwise, the proposition would be to mark GET wait_time as obsolete and to recommand using POST wait_time instead to reduces the chance of PII being logged on anybody's systems.